### PR TITLE
Add upload to pypi from travis tags

### PR DESCRIPTION
**Pull request by juanjux**
Signed-off-by: Juanjo Alvarez <juanjo@sourced.tech>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,6 +17,14 @@ install:
   - pip3 install -r requirements.txt
   - python3 setup.py --getdeps --log
   - pip3 install . --upgrade
+  - if [[ -z "$TRAVIS_TAG" ]]; then exit 0; fi
```
Comment (dennwc) [inline .travis.yml]: Will it actually exit the script? For example, each Docker's `RUN` command executes in a separate script. Might not be the case with Travis, though.

File: .travis.yml [hunk 2]
```diff
@@ -17,6 +17,14 @@ install:
...
+  - echo "[distutils]" > $HOME/.pypirc
```
Comment (dennwc) [inline .travis.yml]: Maybe we can host the header of this file in the repo and add only username/password here?

File: .travis.yml (hunk 1 above)
Comment (juanjux) [reply .travis.yml]: Looks like it does (from this PR CI at least).

File: .travis.yml (hunk 2 above)
Comment (juanjux) [reply .travis.yml]: We could, but it's very small so this makes it one unrelated file less.

**Comment by bzz** [comment]
Would be nice if this could read version from the "${TRAVIS_TAG}"..

**Pull request merged**